### ISSUE TITLE
Summarize VPAT issues not being addressed

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1460]
Annotate excel sheet or write a new sheet that outlines any VPAT issue that is not being addressed because of interference with workflow/user needs.

---

Filled information about fixes into Google sheet.  The geoblacklight changes are the remaining ones to address and there are some we need to discuss in a meeting.
